_posts/2018-09-08-terminal-tips.md

```diff
@@ -95,5 +95,20 @@ Finally to reflect these changes in your remote repository from your local repos
 
 ---
 
+*Contributed by: Ramya CP*
 
+### Reverse search
```

Comment:
Sorry, this tip has already been added before you in #27 

Remove this tip and I'll merge the rest

Reply:
Sure,Thank you!